project/src/components/app/app.tsx

```diff
@@ -1,11 +1,35 @@
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+
 import MainPage from '../../pages/main-page/main-page';
+import Favorites from '../../pages/favorites/favorites';
+import Propepty from '../../pages/property/property';
```

Comment:
```suggestion
import Property from '../../pages/property/property';
```
ниже где используешь - исправить